python/mosaic/api/functions.py

```diff
@@ -208,7 +208,9 @@ def st_bufferloop(
     )
 
 
-def st_buffer_cap_style(geom: ColumnOrName, radius: ColumnOrName, cap_style: ColumnOrName) -> Column:
+def st_buffer_cap_style(
```

Comment:
we should remove this - it does nothing.
the JTS implementation as well, basically does the same normal buffer does :/

Reply:
OK, I've pulled it out. 👍